docs/learn-staking.md

```diff
@@ -104,7 +104,14 @@ Slashing will happen if a validator misbehaves (e.g. goes offline, attacks the n
 
 Validator pools with larger total stake backing them will get slashed more harshly than less popular ones, so we encourage nominators to shift their nominations to less popular validators to reduce the possible losses.
 
-Based on Polkadot's latest codebase, the following slashing conditions have been implemented:
+The following levels of offence are [defined](https://research.web3.foundation/en/latest/polkadot/slashing/amounts.html):
+
+* Level 1: isolated unresponsiveness, i.e. going offline for a long time. No slashing, only [_chilling_](#chilling).
+* Level 2: concurrent unresponsiveness or isolated equivocation. Slashing up to 1% of the staked amount.
+* Level 3: misconducts unlikely to be accidental, but which do not harm the network's security to any large extent. Examples include concurrent equivocation or isolated cases of unjustified voting in [GRANDPA](learn-consensus). Slashing up to 10%.
+* Level 4: misconduct that poses a serious security or monetary risk to the system, or mass collusion. Slashing 100% of the stake behind the offenders.
```

Comment:
These don't exactly line up with the way it's implemented.  We might want to keep the levels but not give the slashing percentages.  This way it's still "conceptually" correct without giving any false info.

Reply:
Changed